frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -6,69 +6,20 @@ import { Helmet } from 'react-helmet';
 import { useTranslation } from 'react-i18next';
 
 import { ClusterDashboard } from './cluster-dashboard/cluster-dashboard';
-import { HorizontalNav, PageHeading, LoadingBox, Page, AsyncComponent } from '../../utils';
-import Dashboard from '@console/shared/src/components/dashboard/Dashboard';
-import DashboardGrid, {
-  GridPosition,
-  GridDashboardCard,
-} from '@console/shared/src/components/dashboard/DashboardGrid';
-import {
-  useExtensions,
-  DashboardsCard,
-  DashboardsTab,
-  isDashboardsCard,
-  isDashboardsTab,
-} from '@console/plugin-sdk';
+import { HorizontalNav, PageHeading, LoadingBox } from '../../utils';
 import { RootState } from '../../../redux';
 
-const getCardsOnPosition = (cards: DashboardsCard[], position: GridPosition): GridDashboardCard[] =>
-  cards
-    .filter((c) => c.properties.position === position)
-    .map((c) => ({
-      Card: () => <AsyncComponent loader={c.properties.loader} />,
-      span: c.properties.span,
-    }));
-
-const getPluginTabPages = (tabs: DashboardsTab[], cards: DashboardsCard[]): Page[] =>
-  tabs.map((tab) => {
-    const tabCards = cards.filter((c) => c.properties.tab === tab.properties.id);
-    return {
-      href: tab.properties.id,
-      name: tab.properties.title,
-      component: () => (
-        <Dashboard>
-          <DashboardGrid
-            mainCards={getCardsOnPosition(tabCards, GridPosition.MAIN)}
-            leftCards={getCardsOnPosition(tabCards, GridPosition.LEFT)}
-            rightCards={getCardsOnPosition(tabCards, GridPosition.RIGHT)}
-          />
-        </Dashboard>
-      ),
-    };
-  });
-
 const DashboardsPage_: React.FC<DashboardsPageProps> = ({ match, kindsInFlight, k8sModels }) => {
   const { t } = useTranslation();
   const title = t('dashboard~Overview');
-  const tabExtensions = useExtensions<DashboardsTab>(isDashboardsTab);
-  const cardExtensions = useExtensions<DashboardsCard>(isDashboardsCard);
 
-  const pluginPages = React.useMemo(() => getPluginTabPages(tabExtensions, cardExtensions), [
-    tabExtensions,
-    cardExtensions,
-  ]);
-
-  const allPages = React.useMemo(
-    () => [
-      {
-        href: '',
-        name: t('dashboard~Cluster'),
-        component: ClusterDashboard,
-      },
-      ...pluginPages,
-    ],
-    [pluginPages, t],
-  );
```

Comment:
Again others might find this useful for adding the dashboard tab next to overview?

Reply:
Should I leave the code as a comment in the file then??

Reply:
ack...